hoot-core-test/src/test/cpp/hoot/core/fourpass/FourPassManagerTest.cpp

```diff
@@ -75,7 +75,7 @@ class FourPassManagerTest : public CppUnit::TestFixture
     OsmMap::resetCounters();
     Settings::getInstance().clear();
 
-    FileUtils::removeDir("test-output/conflate/TileConflatorTest.osm-cache");
+    FileUtils::removeDir("test-output/fourpass/FourPassManagerTest.osm-cache");
```

Comment:
Looks like that was just a copy/paste error from way back that didn't rear it's head until now.  Good catch.